tests/test_alias.rs

```diff
@@ -59,22 +59,6 @@ fn test_alias_legacy_section() {
     "###);
 }
 
-#[test]
-fn test_alias_bad_name() {
-    let test_env = TestEnvironment::default();
-    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
-    let repo_path = test_env.env_root().join("repo");
-
-    let stderr = test_env.jj_cmd_cli_error(&repo_path, &["foo."]);
-    insta::assert_snapshot!(stderr, @r###"
-    error: unrecognized subcommand 'foo.'
-
-    Usage: jj [OPTIONS] <COMMAND>
-
-    For more information, try '--help'.
-    "###);
-}
-
```

Comment:
How does this behave after this PR? Can we keep the test to show the new behavior?

Reply:
I've added it back in, and my less-naive approach actually restores the previous behavior as well.